doc/adr/0011-decline-node.md

```diff
@@ -0,0 +1,21 @@
+# 11. Decline Node Request
+
+Date: 11/03/2021
+
+## Status
+
+Accepted
+
+## Context
+
+We want to be able to decline node requests. The current api logic saves an event on the `nodes` stream for all nodes registered on the network. These nodes appear in the ui's nodes table as requests. So far, the requests that were not accepted would pile up and could not be deleted. When accepted, the nodes receive permissions from the MultiChain and can successfully connect to the system. The opposite of the approval of nodes (which is actually a granting of permissions) would be to revoke the permissions. However, this would not lead to the desired outcome because the nodes would still appear in the list of requests as trying to connect to the network.
```

Comment:
mention mutlichains permssion managment page https://www.multichain.com/developers/permissions-management/
and explain which permission would be granted excactly